lib/tasks/vacancies_to_update.yaml

```diff
@@ -212,3 +213,485 @@ We can offer:
+      <p>Please find an application form and job details on our website: www.gshs.org.uk/vacancies
+      Closing date for applications: Noon, Thursday 10 May 2018.</p>
+      <p>This post is subject to an Enhanced DBS Disclosure and Barred List Check. </p>
+      <p>GEORGE STEPHENSON HIGH SCHOOL<br> SOUTHGATE<br> KILLINGWORTH<br> NEWCASTLE UPON TYNE<br> NE12 6SA<br> <p>
+      <p>Headteacher:  Mr Ian Wilkinson<br> Tel:  0191 216 1115<br> E-mail:  georgestephenson.high@northtyneside.gov.uk</p>
+      <p>GSHS is committed to safeguarding and promoting the welfare of children and young people and expects all staff and volunteers to share this commitment.</p> "
+      subject: 'Physical education'
+      experience: "We would love to hear from you if you have:
+      <ul><li>passion and energy to inspire and motivate students and staff</li>
+      <li>a track record of success in raising standards</li>
+      <li>exceptional interpersonal skills </li>
+      <li>a strong commitment to teamwork</li></ul>"
+      min_pay_scale: 'Main pay range 1'
+      max_pay_scale: 'Upper pay range 3'
+    -
+      slug: 'teacher-of-science'
+      job_title: 'Teacher of Science'
+      job_description: "
+      <p>We are seeking to appoint an enthusiastic and well-qualified Teacher of Science with either a Physics or Chemistry specialism, who has a desire to make a real difference to the learning experiences of young people and to raise attainment in Science. The successful applicant will be required to teach Science throughout Key Stages 3 and 4, including GSCE and A Level classes.</p>
+      <p>Benfield is a school that has been significantly raising achievement each year for the last five years and we are looking for a team player who will be committed to continuing this work with us.</p>
+      <p><em>“Benfield is a good school that has improved considerably since its previous inspection“<br>
+      Teachers have good subject knowledge and use this to support learning well. They are enthusiastic and use some innovative techniques that enthuse students”<br>
+      <strong>(Ofsted May 2015)</strong></em></p>
+      <p>Applications are welcomed from both newly-qualified and experienced teachers.</p>
+      <p>Further details and an application pack, can be found on the council website at www.newcastle.gov.uk or email: lilian.reeson@benfield.newcastle.sch.uk</p>
+      <p>Completed applications should be returned to Mrs L Reeson, PA to Headteacher, either by email or post.<br>
+      Email: lilian.reeson@benfield.newcastle.sch.uk <br>
+      Mrs L Reeson, Benfield School, Benfield Road, Newcastle, NE6 4NU</p>
+      <p>Please note CVs are not accepted.<br>
+      <strong>Closing date:                           9.00am Tuesday 8th May 2018<br>
```

Comment:
Another closing date that could go into `ends_on`.

Reply:
@tahb closing date is the application closing date, whilst ends_on is the job end date.